S3Upload.java

```diff
@@ -34,7 +34,7 @@ private static String readFileContents() {
         String fileContents = "";
         try {
             reader = new BufferedReader(new FileReader(
-                    "C:\\<FMI2>"));
+                    "C:\\example.txt"));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource created by the following constructor: *FileReader*. Execution paths that do not contain closure statements were detected. Ensure the object returned by *FileReader()* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).